flink-end-to-end-tests/flink-queryable-state-test/pom.xml

```diff
@@ -107,6 +107,9 @@
 							<transformers>
 								<transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
 									<mainClass>org.apache.flink.streaming.tests.queryablestate.QsStateClient</mainClass>
+									<manifestEntries>
+										<Add-Opens>java.base/java.util</Add-Opens>
+									</manifestEntries>
```

Comment:
This is a handy alternative in case you run a self-contained jar; you can encode the opens/exports in the jars manifest.
Unfortunately this only works if the you use the `-jar` command; we can't use it for the distribution :cry: